### ISSUE TITLE
Analysis: Second graph

### ISSUE THREAD

**Issue by Ruediger3** [mworion/MountWizzard4#185]
Hi Michel,

short question because I have some doubts about graph 2 (line 1, second from left). The color coding indicates East and West hemisphere bit they are mixed randomly on the plot. Either there is a bug or I do not understand it.
Can you pleas explain to me? Thanks!

Rüdiger

![image](https://user-images.githubusercontent.com/65772654/133154519-1da2f632-8aa6-4751-9080-f4cdb99b5e68.png)

**Comment by mworion** [comment]
Oh this might be a matter of text. This hart does not show where the points are made, but where the error vector points to in relation to the RA axis.
Do you have any idea how to describe it ?

**Comment by Ruediger3** [comment]
Hi Michel,

I am sorry, but I do not understand, what this chart should show. 
You mean the axes show the Az circle? I have spent minutes to understand it, but no success.

Is there any document, which describes the individual charts and how to interpret them? I had noticed also when discussion about our results of the models, there is a lot of uncertainty about the information contained in the charts. There are also different interpretations. 

Cheers
Rüdiger
